Sources/RunCatLocalization/Resources/Dashboard.xcstrings

```diff
@@ -93,6 +111,12 @@
             "value" : "Animal Type Runners"
           }
         },
+        "fr" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Coureurs animaux"
```

Comment:
I would like to capitalize the first letter of each word in the category labels, but would this look unnatural in French? (For example, "Coureurs Animaux".)